src/WingetCreateCLI/UserSettings.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.WingetCreateCLI
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using Microsoft.WingetCreateCLI.Logging;
+    using Microsoft.WingetCreateCLI.Models.Settings;
+    using Microsoft.WingetCreateCLI.Properties;
+    using Newtonsoft.Json;
+    using Sharprompt;
+
+    /// <summary>
+    /// UserSettings configuration class for WingetCreate.
+    /// </summary>
+    public static class UserSettings
+    {
+        private static readonly Lazy<string> SettingsPathLazy = new(() => Path.Combine(Common.LocalAppStatePath, "settings.json"));
+
+        private static readonly Lazy<string> SettingsBackupPathLazy = new(() => Path.Combine(Common.LocalAppStatePath, "settings.json.backup"));
+
+        static UserSettings()
+        {
+            LoadSettings();
+        }
+
+        /// <summary>
+        /// Gets the path to the settings json file.
+        /// </summary>
+        public static string SettingsJsonPath => SettingsPathLazy.Value;
+
+        /// <summary>
+        /// Gets the path to the backup settings json file.
+        /// </summary>
+        public static string SettingsBackupJsonPath => SettingsBackupPathLazy.Value;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to disable telemetry.
+        /// </summary>
+        public static bool TelemetryDisable
```

Comment:
>TelemetryDisable [](http://example.com/codeflow?start=27&length=16)

I believe this should be TelemetryDisabled

Reply:
changed to TelemetryDisabled